app/assets/stylesheets/patterns/_browsing-list.scss

```diff
@@ -20,6 +20,6 @@
 .browsing-list > h1,
 .browsing-list-item {
   display: block;
-  padding: $paddingLarge;  
+  padding: $paddingLarge;
```

Comment:
Name of variable `paddingLarge` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered display, margin, padding